R/class-workbook.R

```diff
@@ -4821,12 +4821,14 @@ wbWorkbook <- R6::R6Class(
     add_form_control = function(
       sheet   = current_sheet(),
       dims    = "A1",
-      type    = NULL,
+      type    = c("Checkbox", "Radio", "Drop"),
       text    = NULL,
       link    = NULL,
       range   = NULL,
       checked = FALSE
     ) {
+      # FIXME The radio button can't be unchecked. (the checkbox works)
+      # TODO add button to many cells? (i.e. dims is larger than 1.)
```

Comment:
Should be possible, but nothing I'm overly interested in. It works reasonably well and if somebody wants other features they should see for themself. Please remove the comment